docs/user-guide/plugins/code-scan.md

```diff
@@ -4,7 +4,7 @@
 The Code Scan plugin of Devtron allows you to perform the code scanning using Trivy. By integrating the **Code Scan** plugin into your workflow you can detect common Vulnerabilities, Misconfigurations, License Risks, and Exposed Secrets in your code.
 
 ### Prerequisites
-No prerequisites are required for integrating **Code Scan** plugin.
+Before integrating the **Code Scan** plugin, install the `Vulnerability Scanning (Trivy/Clair)` integration from Devtron Stack Manager.
```

Comment:
Hyperlink to [Vulnerability Scanning (Trivy/Clair)](https://docs.devtron.ai/usage/integrations/clair) in documentation